### ISSUE TITLE
Paragraph on "We believe that failures make us experts"

### ISSUE THREAD

**Issue by itrich** [SovereignCloudStack/open-operations-manifesto#4]

**Comment by itrich** [comment]
Let's think about merging this item to one of the other paragraphs!

**Comment by fkr** [comment]
Proposal:

```
We firmly believe that failures make us experts. The way we handle mistakes is how we become better. We think, that handling mistakes in an optimal way means to communicate about them openly and without shame. We are convinced, that the culture we build around failures is key for an innovative environment: Not being afraid of making mistakes opens the door to trying new things, without having the fear to fail. We encourage people to raise their voice - not only in case of success, but also in case of failure. We are sure, that this is one of the most best ways to prevent others from making the same mistakes we did.
```

**Comment by fkr** [comment]
Proposal comes from @heinritzkat and @cedegi

**Comment by fkr** [comment]
finetuning in #13

**Comment by fkr** [comment]
@itrich this can be closed as we finalized the PR #13, do you concur?
